plugins/DotCodeMultiline.ts

```diff
@@ -0,0 +1,19 @@
+import visit from 'unist-util-visit';
+import { Node } from 'unist';
+
+interface CustomNode extends Node {
+    value: string;
+    children?: CustomNode[];
+}
+
+export default function () {
+    return function (node: CustomNode): void {
+        visit(node, 'jsx', (node: CustomNode) => {
+            const value = node.value;
+            const preContent = new RegExp(/<pre>(.|\n)*?<\/pre>/gi);
+            node.value = value.replace(preContent, (match) => {
+                return match.replace(/\n/g, '<br />');
+            });
+        });
+    };
+}
```

Comment:
What is this for?

Reply:
That's the plugin to solve the issue that `<pre>` and `<code>` tags had, which made all the code remain on one line.